maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenUtil.java

```diff
@@ -486,6 +498,29 @@ public static boolean isPomFile( IFile pomFile )
             pomFile.getParent() instanceof IProject;
     }
 
+    public static void importProject( String location, IProgressMonitor monitor )
```

Comment:
no , maven plugin project is created by archetype and imported by m2e , there are no code that can be shared .